anitya/tests/lib/backends/test_github.py

```diff
@@ -232,6 +232,19 @@ def test_get_versions(self):
         obs = backend.GithubBackend.get_ordered_versions(project)
         self.assertEqual(obs, exp)
 
+        project.latest_known_cursor = "invalid cursor"
```

Comment:
Could you move this to separate test? I want to break this to multiple tests anyway in the future.

Reply:
Then let me break it up now. I was bothered by this test being so big anyway. :wink: